agreements/activity-types.md

```diff
@@ -0,0 +1,19 @@
+# Activity Types Agreement
+
+We have assessed that there are four types of activities that members carry out within Root Systems:
+
+## External
+
+This is contracting work with the outside world - usually software development, but could include any other types of consulting work.
+
+## Internal Investment
+
+This is either internal work towards development of a Root Systems product, or personal development / upskilling of a member (i.e. conferences, training).
```

Comment:
my understanding is that "internal investment" requires approval from the group. so going to a conference is "personal" unless you ask the group for a mandate to go as "internal investment".